images/linux/scripts/installers/java-tools.sh

```diff
@@ -85,6 +90,22 @@ for cmd in gradle java javac mvn ant; do
     fi
 done
 
+javaVersion=`java -version 2>&1 | head -n 1 | cut -d\" -f 2`
+if [[ "$javaVersion" =~ ([1]{0,1}.)?$DEFAULT_JDK_VERSION.* ]]; then
```

Comment:
Can we use for to remove duplicate string int 95-99?

Reply:
changed to function